src/components/Content/Layout.tsx

```diff
@@ -9,16 +10,21 @@ import { FullPageLoad } from "components/Loading/FullPageLoad";
 import { PageGrid } from "components/styles/Layout";
 import { TaskStatusIconLegend } from "components/TaskStatusIconLegend";
 import WelcomeModal from "components/WelcomeModal";
+import { CY_DISABLE_NEW_USER_WELCOME_MODAL } from "constants/cookies";
 import { size } from "constants/tokens";
 import { newSpruceUser } from "constants/welcomeModalProps";
 import { useAuthStateContext } from "context/auth";
 import { UserQuery, UserQueryVariables } from "gql/generated/types";
 import { GET_USER } from "gql/queries";
 import { useUserSettings } from "hooks";
 import { useAnnouncementToast } from "hooks/useAnnouncementToast";
+import { isProduction } from "utils/environmentVariables";
 
 const { gray, white } = palette;
 
+const disableForTest =
```

Comment:
nit. 
```suggestion
const shouldDisableForTest =
```
or
`isDisabledForTest`